src/analysis/normed_space/ordered.lean

```diff
@@ -1,15 +1,34 @@
 /-
 Copyright (c) 2020 Anatole Dedecker. All rights reserved.
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
```

Comment:
We just use the copyright lines to record the initial creation of the file, but don't add another line for each author.

Reply:
Okay, I've split my changes into a separate file, as per the next comment, so `analysis/normed_space/ordered` is no longer part of this PR.